src/js/close-button.js

```diff
@@ -6,20 +9,55 @@ import Component from './component';
  * when it is activated.
  *
  * @extends Button
- * @class CloseButton
  */
 class CloseButton extends Button {
 
+  /**
+   * The constructor for CloseButton
+   *
+   * @param {Player} player
+   *        the player to attach this button to
+   *
+   * @param {Object} [options]
+   *        Object of option names and values
+   */
   constructor(player, options) {
     super(player, options);
     this.controlText(options && options.controlText || this.localize('Close'));
   }
 
+  /**
+   * Constructs a string to be used for the component elements
```

Comment:
anything that you override needs to be documented, we get anything that is not overridden, in the docs for the object. There are ways to say that this method should use the parents doc comments, but doing so causes jsdoc to link to the parent function rather than the current files function (so it seems like the current file does not implement it)